winch/codegen/src/isa/x64/abi.rs

```diff
@@ -202,7 +207,10 @@ impl X64ABI {
                 ty,
             ),
 
-            ty => unimplemented!("Support for argument of WasmType: {ty}"),
+            ty @ WasmValType::V128 => (
+                Self::vector_reg_for(index_env.next_vr(), call_conv, params_or_returns),
```

Comment:
I could change this to just reuse the existing floating point register methods given this is inside the x64 ABI if that would be preferable.